tests/integration/defs/accuracy/test_llm_api_pytorch.py

```
@@ -5193,6 +5193,180 @@ def test_w4_1gpu_piecewise_cuda_graph(self, mocker):
+                resumed += s.get("iterResumedRequests", 0)
+                pgs = s.get("kvCacheIterationStatsByPoolGroup")
+                if not pgs:
+                    continue
+                for pg in pgs.values():
+                    offload += pg.get("iterOffloadBytes", 0)
+                    onboard += pg.get("iterOnboardBytes", 0)
+            return suspended, resumed, offload, onboard
+
+        def _common_prefix_len(a, b):
+            n = 0
+            for x, y in zip(a, b):
```

Ruff (0.15.21)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)